app/assets/javascripts/components/util/withRouter.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {
+  useLocation,
+  useNavigate,
+  useParams,
+} from 'react-router-dom';
+
+function withRouter(Component) {
```

Comment:
A comment explaining the reason for this new wrapper component, and when (if ever) it should be used with new components, would be helpful. Is this just something we need for wrapping class components, and can get rid when wrapped components get converted to functional components?

Reply:
Done!